index.js

```diff
@@ -0,0 +1,112 @@
+
+const allTripsURL = 'https://ada-backtrek-api.herokuapp.com/trips'
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const listallTrips = () => {
+  reportStatus('Loading something...');
+  const tripsList = $('#tripList')
+  tripsList.empty();
+
+  axios.get(allTripsURL)
+  .then((response) => {
+    reportStatus(`Successfully loaded ${response.data.length} trips`);
+    console.log(response);
+    const result = response.data
+    result.forEach((place) => {
+      tripsList.append(`<li id='${place.id}'>${place.name}</li>`);
+    })
+  })
+  .catch((error) => {
+    reportStatus(`Encountered an error while loading trips: ${error.message}`);
+    console.log(error);
+  });
+
+}; //end of listallTrips
+
+const tripDetails = (trip) => {
+
+  const aboutTrip = $('#details')
+  const singleTripUrl = 'https://ada-backtrek-api.herokuapp.com/trips/'
+  reportStatus('Loading a single something...');
+  console.log(trip);
+
+  axios.get(singleTripUrl + trip.id)
+  .then((response) => {
+    reportStatus(`Successfully loaded ${response.data.name} trip`);
+    console.log(response);
+    const result = response.data
```

Comment:
It seems like on a simple variable assignment like this, you have a tendency to miss the semi-colon. Watch out for this!